tests/backward_comaptibility_tests.sh

```diff
@@ -8,44 +8,41 @@ source ~/.bashrc
 CLUSTER_NAME="test-back-compat-$USER"
 source $(conda info --base 2> /dev/null)/etc/profile.d/conda.sh
 
-git clone https://github.com/skypilot-org/skypilot.git sky-master || true
+git clone https://github.com/skypilot-org/skypilot.git ../sky-master || true
 
 
 # Create environment for compatibility tests
-mamba > /dev/null 2>&1 || conda install -n base -c conda-forge mamba -y
-mamba init
-source ~/.bashrc
-mamba env list | grep sky-back-compat-master || mamba create -n sky-back-compat-master -y python=3.9
+conda env list | grep sky-back-compat-master || conda create -n sky-back-compat-master -y python=3.9
 
-mamba activate sky-back-compat-master
-mamba install -c conda-forge google-cloud-sdk -y
+conda activate sky-back-compat-master
+conda install -c conda-forge google-cloud-sdk -y
 rm -r  ~/.sky/wheels || true
-cd sky-master
+cd ../sky-master
 git pull origin master
 pip install -e ".[all]"
 cd -
 
-mamba env list | grep sky-back-compat-current || mamba create -n sky-back-compat-current -y --clone sky-back-compat-master
-mamba activate sky-back-compat-current
+conda env list | grep sky-back-compat-current || conda create -n sky-back-compat-current -y python=3.9
+conda activate sky-back-compat-current
 rm -r  ~/.sky/wheels || true
 pip uninstall -y skypilot
 pip install -e ".[all]"
 
 
 # exec + launch
 if [ "$start_from" -le 1 ]; then
-mamba activate sky-back-compat-master
+conda activate sky-back-compat-master
```

Comment:
Curious - was there any particular reason we were using mamba here and switched to conda now?